src/dbus/instropect.xml

```diff
@@ -0,0 +1,163 @@
+      <arg name="panid" type="q"/>
+      <arg name="extpanid" type="t"/>
+      <arg name="master_key" type="a{b}"/>
+      <arg name="channel" type="b"/>
+      <arg name="pskc" type="a{b}"/>
+      <arg name="rx_on_when_idle" type="b"/>
+      <arg name="secure_data_requests" type="b"/>
+      <arg name="device_type" type="b"/>
+      <arg name="has_network_data" type="b"/>
+      <arg name="channel_mask" type="u"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="OpenUnsecurePort">
+      <arg name="timeout" type="i"/>
+      <arg name="port" type="q"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="JoinerStart">
+      <arg name="pskd" type="s"/>
+      <arg name="provision_url" type="s"/>
+      <arg name="vendor_name" type="s"/>
+      <arg name="vendor_model" type="s"/>
+      <arg name="vendor_sw_version" type="s"/>
+      <arg name="vendor_data" type="s"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="JoinerStop">
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="Leave">
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="Reset">
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="AddOnMeshPrefix">
+      <arg name="prefix" type="a{b}"/>
+      <arg name="preference" type="b"/>
+      <arg name="preferred" type="b"/>
+      <arg name="slaac" type="b"/>
+      <arg name="dhcp" type="b"/>
+      <arg name="configure" type="b"/>
+      <arg name="default_route" type="b"/>
+      <arg name="on_mesh" type="b"/>
+      <arg name="stable" type="b"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="RemoveOnMeshPrefix">
+      <arg name="prefix" type="a{b}"/>
+      <arg name="preference" type="b"/>
+      <arg name="preferred" type="b"/>
+      <arg name="slaac" type="b"/>
+      <arg name="dhcp" type="b"/>
+      <arg name="configure" type="b"/>
+      <arg name="default_route" type="b"/>
+      <arg name="on_mesh" type="b"/>
+      <arg name="stable" type="b"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="SetMeshLocalPrefix">
+      <arg name="prefix" type="a{b}"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="SetLegacyULAPrefix">
+      <arg name="prefix" type="a{b}"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="SetLinkMode">
+      <arg name="rx_on_when_idle" type="b"/>
+      <arg name="secure_data_requests" type="b"/>
+      <arg name="device_type" type="b"/>
+      <arg name="has_network_data" type="b"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="SetMasterKey">
+      <arg name="master_key" type="a{b}"/>
+      <arg name="err" type="b" direction="out"/>
+    </method>
+
+    <method name="SetPSKc">
```

Comment:
I tend to define these as DBus properties, which will have some standard supporting tools.